src/components/Home/Home.js

```diff
@@ -0,0 +1,275 @@
+        var idEvents = e.target.value;
+        var idEvent = parseInt(idEvents);
+        this.setState({idEvent:idEvent});
+    }
+
+    editButtonAction = (presentArray) => {
+        const editor = this.state.editable;
+        const idEventValue = this.state.idEvent;
+        if(!editor) {
+            return(
+                this.presentArray.map(function (reminder, index) {
+                    if(idEventValue === reminder.id) {
+                        return(
+                            <div key={index}>
+                                <input type="text" className="reminder-add" placeholder={reminder.message} value={this.state.editMessage} onChange={this.editReminderMessage}/>
+                                <input type="number" className="reminder-number" placeholder={reminder.message} value={this.state.editNumber} onChange={this.editNumber} />
+                                <div className="reminder-date">
+                                    <Datetime placeholder="Date & Time" placeholder={reminder.scheduled_datetime} onChange={this.editAddDate}/>
+                                </div>
+                                <button className="save-button" onClick={this.EditButtonValidate}>SAVE</button>
+                                <button className="cancel-button" onClick={this.cancelValue}>CANCEL</button>
+                            </div>
+                        );
+                    } else {
+                        return(
+                            <div key={index}>
+                                <span className="reminder-message"> {reminder.message} at </span>
+                                <span className="reminder-datetime">{moment(reminder.scheduled_datetime ).format("MMMM D, HH:mmA")}</span>
+                                <button className="remove-button" value={reminder.id} onClick={this.handleRemoveAction}>REMOVE </button>
+                                <button className="edit-button" value={reminder.id}>EDIT </button>
+                            </div>
+                        );
+                    }
+                },this)
+            );
+        }
+        else {
+            return(
+            this.presentArray.map(function (reminder, index) {
+                return(
+                    <div key={index}>
+                        <span className="reminder-message"> {reminder.message} at </span>
+                        <span className="reminder-datetime">{moment(reminder.scheduled_datetime ).format("MMMM D, HH:mmA")}</span>
+                        <button className="remove-button" value={reminder.id} onClick={this.handleRemoveAction}>REMOVE </button>
+                        <button className="edit-button" onClick={this.editvalue} value={reminder.id}>EDIT </button>
+                    </div>
+                );
+            },this) 
+        );
+    }
+}
+
+
+
+render() {
+        return (
+            <div className="home-page">
+            {this.handleReminders()}
```

Comment:
`this.handleRemainder()` is not rendering anything right? then why are you passing it braces (`{}`), as I mentioned in my earlier comment, you could have returned past and upcoming remainders from `this.handleRemainders()` and assign it to variable.